### ISSUE TITLE
您好 能否告诉我文件的对应关系？

### ISSUE THREAD

**Issue by linlinx666** [muzishen/IMAGDressing#53]
I noticed that your IGPAIR folder already contains IMAGDressing\IGPair\clothes, IMAGDressing\IGPair\images, IMAGDressing\IGPair\openpose, and IMAGDressing\IGPair\IGPair.json. The evaluation process requires cloth_path, cloth_mask_path, model_path, pose_path, face_path, and text_prompts. Can I understand that, using the mask file, you can generate the cloth_mask with your own model using the cloth? The model_path is the path to the model you ran, pose_path is IMAGDressing\IGPair\openpose, and you haven't provided the face_path, so I would need your help in providing face_path for evaluation. The text_prompts are the IMAGDressing\IGPair\IGPair.json. Can I understand it this way?

**Comment by WhiteJiang** [comment]
@linlinx666 Our project encourages customized testing, so our project's test set only provides clothing images in IGPair_Test. The other folders in IGPair are for training. Of course, if you just want to do a simple evaluation, you can use them without using the training set to train the model. It is worth noting that model_path is the path to the generated model image, not the model path.

**Comment by linlinx666** [comment]
1. 我能不能用 IGpair 的 cloth 文件作为评估所需要的 clothes，能否用 IGpair 的 open pose 作为评估需要的 pose 文件，能否用 IGpair 的 json 文件作为 text prompts，face_path 是昨天您提供的 face 文件，如果可以，那 model_path 应该是什么，里面应该放什么内容，这个内容在哪，从哪里找，
2. 另外，我通过您的代码训练了模型，模型的作用在评估过程中是不是仅仅只是把 cloth 转化为 cloth_mask？



发自我的iPhone


------------------ Original ------------------
From: Xin Jiang ***@***.***&gt;
Date: Tue,Dec 3,2024 9:19 AM
To: muzishen/IMAGDressing ***@***.***&gt;
Cc: linlinx666 ***@***.***&gt;, Mention ***@***.***&gt;
Subject: Re: [muzishen/IMAGDressing] Hello, could you please explain thecorrespondence between the files? (Issue #53)





 
@linlinx666 Our project encourages customized testing, so our project's test set only provides clothing images in IGPair_Test. The other folders in IGPair are for training. Of course, if you just want to do a simple evaluation, you can use them without using the training set to train the model. It is worth noting that model_path is the path to the generated model image, not the model path.
